notebooks/214-vehicle-detection-and-recognition/README.md

```diff
@@ -0,0 +1,19 @@
+# Vehicle Detection And Recognition with OpenVINO
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/IRONICBo/openvino_notebooks/HEAD?labpath=notebooks%2F214-vehicle-detection-and-recognition%2F214-vehicle-detection-and-recognition.ipynb)
+
+<div  align='center' ><img src="data/vehicle-result.png" alt="drawing"/></div>
+
+Thousands of vehicles are running in road.We can detect a single vehicle from a raw image and recognize attributes of vehicles.In this notebook,we use detection model and recognition model to realize the target.
+
+
+## Notebook Contents
+
+In this notebook, we will use both detection model and classification model with OpenVINO.We use [Object Detection Models](https://docs.openvino.ai/2020.2/usergroup3.html) and [Object Recognition Models](https://docs.openvino.ai/2020.2/usergroup4.html) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo).Detection model is used to detect vehicle position.Besides, we crop single vehicle and infer with classification model to recognize attributes of single vehicle.The pipline is here： 
```

Comment:
The links redirect to the old documentation page. It's better to replace them by open model zoo links e.g. https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/vehicle-detection-0200